package.json

```diff
@@ -50,5 +50,8 @@
 		"vest": "^4.6.11",
 		"vite": "^4.0.0"
 	},
-	"type": "module"
-}
\ No newline at end of file
+	"type": "module",
+	"dependencies": {
+		"qrcode": "^1.5.3"
```

Comment:
@IgnacioDebat small detail here!

This should be devDependencie..
Just delete de dependencies and move this package to the list of devDependencies..

Reply:
Mmm not sure about it..
Maybe to make things right, just `npm uninstall` the package, and then `npm install -D` the package again with the `-D` flag..